voyager_archive/templates/voyager_archive/item_display.html

```diff
@@ -0,0 +1,136 @@
+</form>
+<hr/>
+
+
+<div class="row">
+    <div class="column">
+        <h1>Basic and Acquisitions Information</h1>
+        <table class="data-display">
+            <tr>
+                <td>Item Barcode</td>
+                <td>{{ item.item_barcode }}</td>
+            </tr>
+            <tr>
+                <td>Item ID</td>
+                <td>{{ item.item_id }}</td>
+            </tr>
+            <tr>
+                <td>MFHD ID</td>
+                <td>{{ item.mfhd_id }}</td>
+            </tr>
+            <tr>
+                <td>Item Type</td>
+                <td>{{ item.item_type }}</td>
+            </tr>
+            <tr>
+                <td>Temporary Item Type</td>
+                <td>{{ item.temp_item_type }}</td>
+            </tr>
+            <tr>
+                <td>Year</td>
+                <td>{{ item.year }}</td>
+            </tr>
+            <tr>
+                <td>Pieces</td>
+                <td>{{ item.pieces }}</td>
+            </tr>
+            <tr>
+                <td>Price</td>
+                <td>{{ item.price }}</td>
+            </tr>
+            <tr>
+                <td>Caption</td>
+                <td>{{ item.caption }}</td>
+            </tr>
+            <tr>
+                <td>Freetext</td>
+                <td>{{ item.freetext }}</td>
+            </tr>
+            <tr>
+                <td>Item Sequence Number</td>
+                <td>{{ item.item_sequence_number }}</td>
+            </tr>
+            <tr>
+                <td>Chron</td>
+                <td>{{ item.chron }}</td>
+            </tr>
+        </table>
+    </div>
+    <div class="column">
+        <h1>Location and Circulation Information</h1>
+        <table class="data-display">
+            <tr>
+                <td>Permanent Location</td>
+                <td>{{ item.perm_location }}</td>
+            </tr>
+            <tr>
+                <td>Temporary Location</td>
+                <td>{{ item.temp_location }}</td>
+            </tr>
+            <tr>
+                <td>Historical Charges</td>
+                <td>{{ item.historical_charges }}</td>
+            </tr>
+            <tr>
+                <td>Historical Browses</td>
+                <td>{{ item.historical_browses }}</td>
+            </tr>
+            <tr>
+                <td>Holds Placed</td>
+                <td>{{ item.holds_placed }}</td>
+            </tr>
+            <tr>
+                <td>Recalls Placed</td>
+                <td>{{ item.recalls_placed }}</td>
+            </tr>
+            <tr>
+                <td>On Reserve</td>
+                <td>{{ item.on_reserve }}</td>
+            </tr>
+            <tr>
+                <td>Reserve Charges</td>
+                <td>{{ item.reserve_charges }}</td>
+            </tr>
+        </table>
+    </div>
+    <div class="column">
+        <h1>Metadata Information</h1>
+        <table class="data-display">
+            <tr>
+                <td>Metadata Creator ID</td>
```

Comment:
Remove `Metadata` from this set of labels; it's redundant.  Fine to keep it in the table name.